app/src/main/java/com/lagradost/cloudstream3/MainActivity.kt

```diff
@@ -1658,7 +1670,27 @@ class MainActivity : AppCompatActivity(), ColorPickerDialogListener, BiometricCa
                     }
                 }
             }
-            noFocus(this)
+            //noFocus(this)
+
+            val navProfileRoot = findViewById<LinearLayout>(R.id.nav_footer_root)
+            val navProfilePic = findViewById<ImageView>(R.id.nav_footer_profile_pic)
+            val navProfileCard = findViewById<CardView>(R.id.nav_footer_profile_card)
+
+            navProfileCard?.setOnClickListener {
+                showAccountSelectLinear()
+            }
+
+            val profileImage = DataStoreHelper.getCurrentAccount()?.image ?: DataStoreHelper.getDefaultAccount(context).image
```

Comment:
No, why are you doing this here when you have the observe. Remove the homeViewModel.currentAccount shit and do everything here or use observe properly. I would recommend doing it here as accounts are not tied to the home view.

Reply:
I am using the observe to update after the account management dialog and here to update on start up

Reply:
You mean I should observe here to be able to update across all instead of home?

Reply:
Observe updates instantly if you have set the value beforehand. Right now I am just worried that this code can lead to more bugs like this, as the account you observe does not include the default account.

Reply:
So stick to one approach and not both, either make observe the only thing that changes the UI or only change UI with the callback event.